src/components/Avatar.tsx

```diff
@@ -100,7 +100,8 @@ function Avatar({
     let iconColors;
     if (isWorkspace) {
         iconColors = StyleUtils.getDefaultWorkspaceAvatarColor(avatarID?.toString() ?? '');
-    } else if (useFallBackAvatar) {
+        // Assign the icon fill color only for the default fallback avatar
+    } else if (useFallBackAvatar && fallbackIcon === Expensicons.FallbackAvatar) {
```

Comment:
@twilight2294 here we should not check only `fallbackIcon`, bcoz for other cases also it can be FallbackAvatar. Let's update that condition to `avatarSource === Expensicons.FallbackAvatar` to fix this DB https://github.com/Expensify/App/issues/69697